### ISSUE TITLE
Click targets for axis contraint on 'compass rose' are very small

### ISSUE THREAD

**Issue by 0HyperCube** [GraphiteEditor/Graphite#2312]
To drag constrained to an axis, it is necessary to have very precise motor control or to zoom the browser in. Since even I can't do it reliably, this isn't really great for accessibility. Perhaps the size of the target should be increased or the functionality removed?

There are other easier ways to accomplish the same thing (drag with <kbd>shift</kbd> or use <kbd>G</kbd> then <kbd>X</kbd>) so this isn't particularly high priority.
